doc/sphinx/addendum/type-classes.rst

```diff
@@ -138,21 +141,25 @@ the constraints as a binding context before the instance, e.g.:
 These instances are used just as well as lemmas in the instance hint
 database.
 
+.. _contexts:
+
 Sections and contexts
 ---------------------
 
-To ease the parametrization of developments by typeclasses, we provide a new
-way to introduce variables into section contexts, compatible with the implicit
-argument mechanism. The new command works similarly to the :cmd:`Variables`
-vernacular, except it accepts any binding context as argument. For example:
+To ease the parametrization of developments by typeclasses, one can use
+the command :cmd:`Context` to introduce variables into section contexts,
+it works similarly to the :cmd:`Variable` vernacular, except it accepts any
+binding context as argument, so variables can implicit, and
+:ref:`implicit-generalization` can be used (see also :ref:`section-mechanism`).
+For example:
 
 .. coqtop:: all
```

Comment:
As can be seen in https://coq.gitlab.io/-/coq/-/jobs/179057641/artifacts/_install_ci/share/doc/coq/sphinx/html/addendum/type-classes.html#sections-and-contexts, the example shows the generated obligations, which is not very useful given that they are admitted. We could split this `.. coqtop::` block in three parts, with the intermediate part using option `in` instead of `all` to avoid displaying Coq's answer.